.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v4
+        with:
+          go-mod-file: 'go.mod'
+          cache: false
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
+          args: '-c .github/.golangci.yml'
+          skip-cache: true
```

Comment:
Why are `skip-cache: true` and `cache: false` above required?

Reply:
They're not required, but there's no real benefit to caching here considering the whole thing takes 9 seconds to run.